Tests/InstrumentationTests/InstrumentationSystemTests.swift

```diff
@@ -19,7 +19,7 @@ final class InstrumentationSystemTests: XCTestCase {
     func testItProvidesAccessToASingletonInstrument() {
         let tracer = FakeTracer()
 
-        InstrumentationSystem.bootstrap(tracer)
+        InstrumentationSystem.bootstrap(tracer) // FIXME, must use bootstrapInternal
```

Comment:
fixed now

Reply:
Is there a reason that this still uses `bootstrap` instead of `bootstrapInternal`?